SPDX-License-Identifier: AGPL-3.0-only

Use OpenBSD-style default configuration and log directories for smp-server.

Index: apps/smp-server/Main.hs
--- apps/smp-server/Main.hs.orig
+++ apps/smp-server/Main.hs
@@ -7,10 +7,10 @@ import Simplex.Messaging.Server.Web (ser
 import SMPWeb (smpGenerateSite)
 
 defaultCfgPath :: FilePath
-defaultCfgPath = "/etc/opt/simplex"
+defaultCfgPath = "/etc/simplex/smp-server"
 
 defaultLogPath :: FilePath
-defaultLogPath = "/var/opt/simplex"
+defaultLogPath = "/var/simplex/smp-server"
 
 logCfg :: LogConfig
 logCfg = LogConfig {lc_file = Nothing, lc_stderr = True}
